Project.toml

```diff
@@ -23,22 +23,24 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
 CSV = "0.8.5, 0.9, 0.10"
+Dates = "1.6"
 FilePaths = "0.8.3"
 Glob = "1.3"
 Graphs = "1"
 InfrastructureModels = "0.7.3, 0.7.5"
-Ipopt = "0.9, 1.0.2, 1.1"
+Ipopt = "1"
 JSON = "0.18, 0.19, 0.20, 0.21"
-JuMP = "0.22, 0.23, 1"
+JuMP = "1.23.2"
 LoggingExtras = "0.4.7, 1"
 PolyhedralRelaxations = "0.3.5"
 SCS = "0.9, 1.0, 1.1"
+SparseArrays = "1.6"
 SpecialFunctions = "2"
+Test = "1.6"
```

Comment:
Is there a reason Dates, SparseArrays and Test were added to compat? My impression was that these don't need to be explicitly listed because of the julia compat entry.